src/locales/fr.json.ts

```diff
@@ -44,12 +44,12 @@ exportAsMessages('_locales/fr/messages.json', {
   options_importBlacklistDialog_title: 'Import',
   options_importBlacklistDialog_fromFile: 'Importer depuis un fichier',
   options_importBlacklistDialog_selectFile: 'Sélectionner un fichier',
-  options_importBlacklistDialog_fromPB: 'Importer depuis une Blocklist personnelle',
+  options_importBlacklistDialog_fromPB: 'Importer depuis une Blacklist personnelle',
```

Comment:
There is a confusing mix between Blacklist & Blocklist in the en translations :
```
  // The label for the select to import from Personal Blocklist.
  options_importBlacklistDialog_fromPB: 'Import from Personal Blocklist',
```
Don't know which one should be used.

Reply:
"Personal Blocklist" is a proper noun, so I think it is better to leave it as it is.